pymatviz/ptable.py

```diff
@@ -1425,9 +1425,7 @@ def ptable_hists(
     plot_kwargs: dict[str, Any] | None = None,
     # Axis-scope
     ax_kwargs: dict[str, Any] | None = None,
-    child_kwargs: dict[str, Any]
-    | Callable[[Sequence[float]], dict[str, Any]]
```

Comment:
I'm a bit unsure about `child_kwargs` as `Callable` here

Reply:
i was confused by the `Callable` type in there as well. i think we can remove it. what was the intended use case anyway? allow people to customize `child_kwargs` based on the data for each element tile?

Reply:
> i was confused by the `Callable` type in there as well. i think we can remove it. what was the intended use case anyway? allow people to customize `child_kwargs` based on the data for each element tile?

Not sure, I think this is some legacy from previous implementation of `ptable_hists`? Let's remove it for now, maybe add it when we have a solid use case in the future